block/src/Pos/Block/Lrc.hs

```diff
@@ -76,7 +75,6 @@ type LrcModeFull ctx m =
     , MonadSlots ctx m
     , MonadBlockApply ctx m
     , MonadReader ctx m
-    , SscMessageConstraints
```

Comment:
These `type` constraint sets like, `LrcModeFull` are mostly pretty bonkers and many of the constraints within them could be removed. For instance many of these `Mode` types had `Bi X` as part of them, but when the `Bi` instance as put with the `X` type declaration, the constraints could disappeas.

In the case of this `SscMessageConstraint` is just establishing that these different types have these instances. Since we reunited the instances, `SscMessageConstraint` may not be necessary at all.